package.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
No. The examples/recommended configs used `npm clean-install` for installation of deps. This command requires it. But `npm install` doesn't.  The diff is that it might choose "untested" combination of deps. 

With `.releaserc`we would probably install it directly, e.g.  something like: `npm install -g semantic-release @semantic-release/{exec,git,github,changelog}`

Reply:
I like that option. Lock files in repositories are not my favorite thing to manage.

Reply:
The PR was updated with the `.releaserc` proposal. It required to move the commit lint config to `commitlint.config.js`. I used there the more generic rules set `@commitlint/config-conventional` instead of the one from Angular project. With that the `CONTRIBUTING.md` was also updated.